lib/mozart_fetcher/component.ex

```diff
@@ -1,5 +1,6 @@
 defmodule MozartFetcher.Component do
-  alias MozartFetcher.{Component, Config, Envelope, LocalCache}
+  alias   MozartFetcher.{Component, Config, Envelope, LocalCache}
```

Comment:
is this from the changes of running `mix format`? I think it'd be good to keep using the format provided by `mix format`

Reply:
Sure ill run that